### PR TITLE
174 pylint alerts corrections

### PR DESCRIPTION
**Pull request by evidencebp**
This PR takes care of issue "Pylint alerts corrections as part of intervention experiment" #174.

Each commit takes care of a single Pylint alert in a single file.
See the [executed interventions](https://github.com/evidencebp/pylint-intervention/blob/main/interventions/stanford-oval_storm_interventions_September_22_2024.csv).
The commit message explain the intervention and its justification (e.g., when narrowing an exception).

Most interventions are minor.
The larger ones are due to too-many-branches or too-many-statements.
In these cases I extracted methods, making the code shorter, easier to understand and modify.

When going over the PR please note:
- I noticed some misalignments to black in the existing code. Since it was that way and I wanted to make small and focused interventions, I did not fix them.
- I'm not familiar with the project logic. All interventions are small refactors that should keep the same logic. However, please consider my infamilarity.
- I do not have the needed API keys. Hence, when running the code after the modification the flow did not reach the main path. I run the modified parts externally.
- The conda installation I used did not install fastchat, used in eval\evaluation_prometheus.py . I tried to install it separately yet it did not work. Hence I could not run this file. However the intervention there was replacing  Exception in ValueError, more focused. However, no other exception is expected there so this refactor should be safe.

Please tell me if there are unclear points or some other needed aspects.

### CODE REVIEW

File: frontend/demo_light/storm.py [hunk 1]
```diff
@@ -35,13 +35,15 @@ def main():
     menu_container = st.container()
     with menu_container:
         pages = ["My Articles", "Create New Article"]
+        styles={
```
[the remark this answers is not in the document]
Comment (evidencebp) [reply frontend/demo_light/storm.py:38]: I think that I got this formatting from black.
Should I change it to
styles={ "container": {"padding": "0.2rem 0",
                                                       "background-color": "#22222200"},}
?
Any other formatting?

**Comment by shaoyijia** [comment]
Hi @evidencebp, we recently have a very large update of our repo. Is it possible for you to rerun your algorithm to gets the edit and open a new PR?

**Comment by evidencebp** [comment]
Indeed a large update.
I merged the main into this branch.
Now there are only 3 files with interventions.
At list it makes the review easier...

Can you approve and merge now?

**Review by shaoyijia — APPROVED** [review]
Hope this helps your research!

**Comment by evidencebp** [comment]
Sure it will help!

In case that you would like to intervene yourself in other projects or have ideas for such, please tell me.
Of course, if you know others that might be interested, let them know too.

**Pull request merged**